tests/test_k10_http2_enable.py

```
@@ -0,0 +1,227 @@
+            + b"Connection: close\r\n\r\n"
+            + EXPECTED_BODY
+        )
+        self.transport.write(response)
+        self.transport.close()
+
+    def _receive_http2(self, data: bytes) -> None:
+        if self.transport is None:
+            return
+        try:
+            received_events = self.h2_connection.receive_data(data)
+        except Exception:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)